### PR TITLE
MPI_Wtime wrapper update

### PR DESCRIPTION
**Pull request by jiamingz9925**
Let MPI_Wtime wrapper directly call wtime() and return the result.

### CODE REVIEW

File: mpi-proxy-split/mpi-wrappers/mpi_wrappers.cpp [hunk 1]
```diff
@@ -108,11 +108,19 @@ USER_DEFINED_WRAPPER(int, Init_thread, (int *) argc, (char ***) argv,
 // FIXME: See the comment in the wrapper function, defined
 // later in the file.
 // DEFINE_FNC(int, Finalize, (void))
-DEFINE_FNC(double, Wtime, (void))
 DEFINE_FNC(int, Finalized, (int *) flag)
 DEFINE_FNC(int, Get_processor_name, (char *) name, (int *) resultlen)
 DEFINE_FNC(int, Initialized, (int *) flag)
```
Comment (gc00) [inline mpi-proxy-split/mpi-wrappers/mpi_wrappers.cpp:114]: I think you need to add:
`#include <time.h>`
at the top of the file (above `#include mpi.h`).

**Review by gc00 — COMMENTED** [review]
Pls add the '#include <time.h>'.  I suspect that you're seeing a compiler warning right now, because of it.

File: mpi-proxy-split/mpi-wrappers/mpi_wrappers.cpp (hunk 1 above)
Comment (jiamingz9925) [reply mpi-proxy-split/mpi-wrappers/mpi_wrappers.cpp:115]: Yes, have it added.

**Review by gc00 — APPROVED** [review]
Looks good.  Please push it in.  Thanks.

**Pull request merged**